plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -107,8 +117,12 @@ def deploy_yaml
       @deploy_yaml ||= DeployYaml.new(self)
     end
 
-    def resource_running?(resource)
-      extension_client.send("get_#{deploy_yaml.resource_name}", resource.metadata.name, resource.metadata.namespace)
+    def deployment
+      extension_client.send(
```

Comment:
Couldn't this just be `extension_client.get_deployment`, rather than using `#send`?